addon/templates/components/bgr-breadcrumbs-item.hbs

```diff
@@ -1,4 +1,4 @@
-{{#each bgrBreadcrumbs.instances as |instance|}}
+{{#each breadcrumbsService.instances as |instance|}}
```

Comment:
Unrelated as well, but maybe alias this in the component? Now you had to change the template code due to a service rename.